src/ceph-volume/ceph_volume/util/device.py

```diff
@@ -172,6 +173,22 @@ def json_report(self):
         output['lvs'] = [lv.report() for lv in self.lvs]
         return output
 
+    def _get_device_id(self):
```

Comment:
I would suggest adding a comment explaining the fact we mimic the blkdev implementation so IDs collected in the inventory can be used in inside ceph.

Reply:
Right...lets see where this is going in regards to duplicate implementations.